features/app/native/navigationHandlers.js

```diff
@@ -0,0 +1,49 @@
+import Config from '../../../config';
```

Comment:
I just don't get it why features/app (1) implements an App component which has a property config & (2) at the same time defines a feature-private function which deals with some other (global) config...